### PR TITLE
feat: Enable toggling enabled mods 

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Depends on #28

### CODE REVIEW

File: src-vue/src/views/ModsView.vue [hunk 1]
```diff
@@ -44,6 +45,39 @@ export default defineComponent({
...
+                game_path: this.$store.state.game_path,
+                install_type: this.$store.state.install_type
+            } as GameInstall;
+
+            // enable/disable specific mod
+            try {
+                await invoke("set_mod_enabled_status_caller", {
+                    gameInstall: game_install,
+                    modName: mod.name,
+                    isEnabled: !mod.enabled, // TODO this might set it to the old state
```
Comment (Alystrasz) [inline src-vue/src/views/ModsView.vue:64]: Still TODO? :)

Comment (GeckoEidechse) [reply src-vue/src/views/ModsView.vue]: Whoops, fixed in ab50b171f735ad8cebc390ec84f8da6aa8208dfb

**Comment by Alystrasz** [comment]
Looks like I cannot toggle my mods 😢 

image

**Comment by GeckoEidechse** [comment]
@Alystrasz works for me on two different machines. You sure you checked out the right branch? Maybe local copy was out-of-date? ^^

**Comment by Alystrasz** [comment]
Wrong branch indeed...

**Pull request merged**